src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -654,6 +654,9 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 						new InputSlot(4, "slot2", 20, 30, Color.white, !_true, !_true, new Procedure("procedure4"),

```

Comment:
```suggestion
						new InputSlot(3, "slot2", 20, 30, Color.white, !_true, !_true, new Procedure("procedure4"),
```

I'd change the ID here to avoid slots with conflicting IDs